src/graph/graphcontext.c

```diff
@@ -121,6 +121,18 @@ LabelStore* GraphContext_AddRelationType(GraphContext *gc, const char *label) {
   return store;
 }
 
+const char* GraphContext_GetNodeLabel(const GraphContext *gc, NodeID id) {
+    int label_id = Graph_GetNodeLabel(gc->g, id);
+    if (label_id == GRAPH_NO_LABEL) return NULL;
+    return gc->node_stores[label_id]->label;
+}
+
+const char* GraphContext_GetEdgeRelationType(const GraphContext *gc, Edge *e) {
```

Comment:
It's strange that GraphContext_GetNodeLabel takes NodeID while GraphContext_GetEdgeRelationType takes an Edge pointer, I think it would be better if GraphContext_GetNodeLabel would accept a Node pointer.

Reply:
:+1: